### PR TITLE
zoho mail

### PR DESCRIPTION
**Pull request by EnderPoint07**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x ] The file is in the `domains` folder and is in the JSON format.
- [x ] You have completed your website. 
- [x ] The website is reachable.  
- [x ] You're not using Vercel or Netlify.  
- [x ] The CNAME record doesn't contain `https://` or `/`.  
- [x ] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://enderpoint.is-a.dev

### CODE REVIEW

File: domains/www.enderpoint.json
Comment (WilliamDavidHarrison) [inline domains/www.enderpoint.json:1]: We do not allow domains starting with `www`.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/www.enderpoint.json
Comment (EnderPoint07) [reply domains/www.enderpoint.json:1]: oh

File: domains/enderpoint.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
         "instagram": "enderpoint07"
     },
     "record": {
-        "CNAME": "enderpoint07.github.io"
+        "CNAME": "enderpoint07.github.io",
+        "TXT": "zoho-verification=zb07670443.zmverify.zoho.in"
```
Comment (WilliamDavidHarrison) [inline domains/enderpoint.json:12]: We don't support TXT verification or allow other records with CNAME records.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/enderpoint.json (hunk 1 above)
Comment (EnderPoint07) [reply domains/enderpoint.json:12]: oh

**Review by wdhdev — CHANGES_REQUESTED** [review]
We don't support TXT verification.

File: domains/mail.enderpoint.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "EnderPoint's mail",
+    "owner": {
+        "username": "EnderPoint07",
+        "email": "",
+        "discord": "DotM38#5497",
+        "instagram": "enderpoint07"
+    },
+    "record": {
+        "TXT": "It is what it is"
```
Comment (WilliamDavidHarrison) [inline domains/mail.enderpoint.json:10]: Please use a valid record.

File: domains/zb18471962.mail.enderpoint.json
Comment (WilliamDavidHarrison) [inline domains/zb18471962.mail.enderpoint.json:1]: We do not support domain verification.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**